### DISCUSSION TITLE
Nomad support?

### DISCUSSION THREAD

**Discussion by nahsi** [openebs/openebs#3537] [Ideas] [answered]
Does openebs adhere to CSI standard?

At the first glance all the components are just containers that can be run by any means (Nomad/docker-compose etc).

If so would you welcome documentation refresh that will make documentation more neutral and welcoming for the users of alternative schedulers, remove heavy focus/lock on K8S and add Nomad related instruction and guides?

**Comment by niladrih** [comment, marked as answer]
Hi @nahsi. All of the OpenEBS storage engines do not adhere to the CSI spec. LocalPV-Provisioner (Hostpath) and NFS Provisioner use the out-of-tree external provisioner lib. The rest of the storage engines comply with CSI. However. All of the storage engines require kubernetes, and they haven't been built for use outside of kubernetes.
